xivo_dao/resources/func_key/model.py

```diff
@@ -1,31 +1,22 @@
 # -*- coding: utf-8 -*-
-# Copyright 2014-2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2014-2021 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 
 class Hint(object):
 
-    __slots__ = ['user_id', 'extension', 'argument']
```

Comment:
Why did you decide to remove this? Why was it there in the first place?

Reply:
Re-added thanks